sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -261,6 +261,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val DISK_TO_MEMORY_SIZE_FACTOR = buildConf(
+    "org.apache.spark.sql.execution.datasources.fileDataSizeFactor")
```

Comment:
similar to `spark.sql.sources.parallelPartitionDiscovery.parallelism`, how about `spark.sql.sources. fileDataSizeFactor`

Reply:
shouldn't we call this something like compressionFactor?

Reply:
ah `compressionFactor` sounds better.